src/lib/find-files.ts

```diff
@@ -73,7 +76,9 @@ export async function find(
         found.push(fileFound);
       }
     }
-    return found;
+    const filteredFiles = filterForDefaultManifests(found);
```

Comment:
Can be squashed into 1 line
```suggestion
      return filterForDefaultManifests(found);
```